evm-contracts/src/v0.6/dev/SimpleWriteAccessController.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity ^0.6.0;
+
+import "./SimpleReadAccessController.sol";
+
+/**
+ * @title SimpleWriteAccessController
+ * @notice Allows the owner to set access for addresses. External accounts are
+ * not granted special access.
+ */
+contract SimpleWriteAccessController is SimpleReadAccessController {
```

Comment:
Unfortunately, naming here seems slightly off to me. Let me suggest an alternative I hope you will like :)

As these contracts only differ in that `SimpleReadAccessController` enables access only for external accounts, and `SimpleWriteAccessController` enables access for all approved accounts, it would be my intuition to call them `ExternalAccessController` and `SimpleAccessController`. In both cases, access could be for reads or writes.

Also, I would probably make `SimpleAccessController` the base contracts, and then extend it with `ExternalAccessController` to add the `tx.origin` check.